src/main/java/utils/DateTimeUtils.java

```diff
@@ -0,0 +1,23 @@
+package utils;
+
+import java.time.LocalDate;
+import java.time.YearMonth;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class DateTimeUtils {
+
+    public static String getTimeRelation(String monthYear, LocalDate localDate) {
```

Comment:
Just a friendly reminder: when dealing with date/time values, avoid passing raw strings to represent them. It's unclear what date format you're expecting to parse.(same for getDatePickerValue)